src/panels/lovelace/components/hui-image.ts

```diff
@@ -197,7 +216,8 @@ class HuiImage extends LitElement {
 
   static get styles(): CSSResult {
     return css`
-      img {
+      img,
+      video {
```

Comment:
This is slightly weird, now we are leaking implementation details of our component 🤔